### PR TITLE
Quickstart detection and readme update 

### PR DESCRIPTION
**Pull request by oferbaratz**

### CODE REVIEW

**Comment by oferbaratz** [comment]
README.md updates
Quickstart for detection notebook (examples)

File: src/super_gradients/examples/SG_quickstart_detection.ipynb [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+        "Downloading COCO128 dataset (It is a small portion of the COCO dataset for)"
+      ],
+      "metadata": {
+        "id": "dZapleFh0_GN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# COCO128 dataset https://www.kaggle.com/ultralytics/coco128 (first 128 images from COCO train2017) by Ultralytics\n",
```
Comment (lotem-deci) [inline src/super_gradients/examples/SG_quickstart_detection.ipynb:102]: @oferbaratz  - This needs to be removed from here...

File: src/super_gradients/examples/SG_quickstart_detection.ipynb [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+      "metadata": {
+        "id": "q4f2mezExv3j"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#@title Unziping and adding paths to all images of train and vlaidation sets\n",
```
Comment (lotem-deci) [inline src/super_gradients/examples/SG_quickstart_detection.ipynb:114]: @oferbaratz  - Again - probably residues that should be removed

File: src/super_gradients/examples/SG_quickstart_detection.ipynb [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+        "pycharm": {
+          "name": "#%%\n"
+        },
+        "id": "5nUNT4qsOcOL"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# WE NOW TUNE THE 3 NORMALIZERS ACCORDING TO THE NEW DATASET ATTRIBUTES,\n",
```
Comment (lotem-deci) [inline src/super_gradients/examples/SG_quickstart_detection.ipynb:272]: @oferbaratz  - Not sure what this step is for... Let's go over this together

**Review by lotem-deci — APPROVED** [review]
LGTM

**Pull request merged**